tests/profiling/collector/test_threading.py

```diff
@@ -215,56 +215,57 @@ def test_lock_release_events():
     assert event.sampling_pct == 100
 
 
-@pytest.mark.skipif(not TESTING_GEVENT, reason="Not testing gevent")
-def test_lock_gevent_tasks():
-    r = recorder.Recorder()
-
-    def play_with_lock():
-        lock = threading.Lock()
-        lock.acquire()
-        lock.release()
-
-    with collector_threading.ThreadingLockCollector(r, capture_pct=100):
-        t = threading.Thread(name="foobar", target=play_with_lock)
-        t.start()
-        t.join()
-
-    assert len(r.events[collector_threading.ThreadingLockAcquireEvent]) >= 1
-    assert len(r.events[collector_threading.ThreadingLockReleaseEvent]) >= 1
-
-    for event in r.events[collector_threading.ThreadingLockAcquireEvent]:
-        if event.lock_name == "test_threading.py:223":
-            assert event.thread_id == nogevent.main_thread_id
-            assert event.wait_time_ns >= 0
-            assert event.task_id == t.ident
-            assert event.task_name == "foobar"
-            # It's called through pytest so I'm sure it's gonna be that long, right?
-            assert len(event.frames) > 3
-            assert event.nframes > 3
-            assert event.frames[0] == (__file__.replace(".pyc", ".py"), 224, "play_with_lock", "")
-            assert event.sampling_pct == 100
-            assert event.task_id == t.ident
-            assert event.task_name == "foobar"
-            break
-    else:
-        pytest.fail("Lock event not found")
-
-    for event in r.events[collector_threading.ThreadingLockReleaseEvent]:
-        if event.lock_name == "test_threading.py:223":
-            assert event.thread_id == nogevent.main_thread_id
-            assert event.locked_for_ns >= 0
-            assert event.task_id == t.ident
-            assert event.task_name == "foobar"
-            # It's called through pytest so I'm sure it's gonna be that long, right?
-            assert len(event.frames) > 3
-            assert event.nframes > 3
-            assert event.frames[0] == (__file__.replace(".pyc", ".py"), 225, "play_with_lock", "")
-            assert event.sampling_pct == 100
-            assert event.task_id == t.ident
-            assert event.task_name == "foobar"
-            break
-    else:
-        pytest.fail("Lock event not found")
+# TODO: Requires manual gevent patching in subprocess
```

Comment:
Which subprocess are we talking about in this context?

Reply:
The idea is to re-implement these tests as `@pytest.mark.subprocess` tests where we call `gevent.monkey.patch_all()`. These tests don't work anymore because we have removed the early import and call to `patch_all` from the `sitecustomize`. So we need to do that at some point. However, this might actually overlap with what you've been working on regarding gevent support testing, so we might rely on that and avoid duplication of effort.

Reply:
@P403n1x87 what's the command you use to run this test locally?

Reply:
Fixed! https://github.com/DataDog/dd-trace-py/pull/4863/commits/da7bf58e30abee636cb46343b13b6e5db86eca1f

For future spelunkers, the answer to my question above is `tox -v -e py310-profile-minreqs-gevent -- -k "test_lock_gevent_tasks" -s`